### PR TITLE
etcdctl: backup tool should use the new layout

### PR DESCRIPTION
**Pull request by xiang90**
fix part one of https://github.com/coreos/etcd/issues/2664

### CODE REVIEW

File: etcdctl/command/backup_command.go [hunk 1]
```diff
@@ -44,10 +44,10 @@ func NewBackupCommand() cli.Command {
 
 // handleBackup handles a request that intends to do a backup.
 func handleBackup(c *cli.Context) {
-	srcSnap := path.Join(c.String("data-dir"), "snap")
-	destSnap := path.Join(c.String("backup-dir"), "snap")
-	srcWAL := path.Join(c.String("data-dir"), "wal")
-	destWAL := path.Join(c.String("backup-dir"), "wal")
+	srcSnap := path.Join(c.String("data-dir"), "member/snap")
```
Comment (yichengq) [inline etcdctl/command/backup_command.go]: `"member", "snap"` seems more elegant.

**Comment by yichengq** [comment]
lgtm

i am thinking that we should have example workflow testing. It is disappointing that the workflow in the doc is broken unknown.

**Comment by barakmich** [comment]
lgtm, +1 yichengq's comment

**Comment by xiang90** [comment]
@barakmich @yichengq Fixed. Merging. Thanks.

**Pull request merged**